src/Provider/Youtube/Provider.php

```diff
@@ -148,6 +149,16 @@ public function provide($input)
 
         $response = $this->getHttpClient()->send($request, $options);
 
+        /* Try alternative URL if the status is fail */
```

Comment:
We shouldn't need this second request if we just use the url without `&el=detailpage`